.github/workflows/main.yml

```diff
@@ -52,7 +52,8 @@ jobs:
       fail-fast: false
       matrix:
         ruby:
-          - 3.0
+          - 3.1
+          - '3.0'
```

Comment:
This is a problem for all entries, no? Let's make them all strings.

Reply:
Nope.  It's only a problem for ones that end in zeroes.

3.0 -> 3
'3.0' -> 3.0

That said, I'm happy to make them all strings if you'd like.

Reply:
It looks nicer, but I don't care enough.

Reply:
I rolled it in to the 2.4 change.